src/logic/configure.ts

```diff
@@ -17,10 +18,48 @@ import {
 } from '../common/document';
 import { userError } from '../common/error';
 import { fetchProviderInfo } from '../common/http';
-import { readFile } from '../common/io';
+import { exists, readFile } from '../common/io';
 import { formatShellLog, LogCallback } from '../common/log';
 import { OutputStream } from '../common/output-stream';
+import { envVariable } from '../templates/env';
 
+export async function updateEnv(
+  provider: string,
+  securitySchemes: SecurityScheme[],
+  options?: { warnCb?: LogCallback }
+): Promise<void> {
+  let envContent = '';
+  //Get .env file
+  if (await exists('.env')) {
+    envContent = (await readFile('.env')).toString();
+  }
+  //Get security values of installed provider
+  const envProviderName = provider.replace('-', '_').toUpperCase();
+
+  for (const scheme of securitySchemes) {
+    let value: string | undefined;
+    if (isApiKeySecurityScheme(scheme)) {
+      value = envVariable(`${envProviderName}_API_KEY`, '');
+    } else if (isBasicAuthSecurityScheme(scheme)) {
+      value = envVariable(`${envProviderName}_USERNAME`, '');
+      value = envVariable(`${envProviderName}_PASSWORD`, '');
+    } else if (isBearerTokenSecurityScheme(scheme)) {
+      value = envVariable(`${envProviderName}_TOKEN`, '');
+    } else if (isDigestSecurityScheme(scheme)) {
+      value = envVariable(`${envProviderName}_DIGEST`, '');
+    } else {
+      options?.warnCb?.(
+        `⚠️  Provider: "${provider}" contains unknown security scheme`
+      );
+    }
```

Comment:
You're reassigning templated string under `isBasicAuthSecurityScheme(scheme)` condition. Maybe new function that would encapsulate these if statements in env template module would be cool.

```suggestion
  for (const scheme of securitySchemes) {
    const value = composeEnvVariables(scheme, envProviderName)
    
    if (value === undefined) {
      options?.warnCb?.(
        `⚠️  Provider: "${provider}" contains unknown security scheme`
      );
    }
```